<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neureset</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>381</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#textBrowser{ 
	border-style: solid;
	border-width: 3px;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="sessionFrame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>551</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{ 
	border-style: solid;
	border-width: 3px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <widget class="QLabel" name="sessionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Session Panel</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>531</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="sessionProgress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>133</y>
       <width>531</width>
       <height>20</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="sessionPlay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sessionPause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sessionStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="sessionTimer">
     <property name="geometry">
      <rect>
       <x>323</x>
       <y>20</y>
       <width>211</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="patientFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_2{ 
	border-style: solid;
	border-width: 3px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <widget class="QPushButton" name="patientNewSessionButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>New Session</string>
     </property>
    </widget>
    <widget class="QLabel" name="patientIDLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>64</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Patient ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="patientID">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Log History</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="testFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_3{ 
	border-style: solid;
	border-width: 3px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Panel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chargeBattery">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Charge Battery</string>
     </property>
    </widget>
    <widget class="QPushButton" name="depleteBattery">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Deplete Battery</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>83</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>83</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="power">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTime">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>12</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="batteryStatus">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>12</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="contact">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1e3a8a;
border: 1px solid black;
border-radius: 5px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="treatment_signal">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #14532d;
border: 1px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="contact_lost">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #7f1d1d;
border: 1px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuNeureset">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionDevice"/>
    <addaction name="actionHistory"/>
   </widget>
   <addaction name="menuNeureset"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDevice">
   <property name="text">
    <string>Device</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
